### ISSUE TITLE
Breaking change by goodreads

### ISSUE THREAD

**Issue by gartkb** [rhollister/goodreads#30]
Hi, I guess you might know by now, but just in case you don't - 
Goodreads seems to have made some kind of change so the addon doesn't work on the book page.
The addon continues to work on book lists pages.
Some number of people reporting this on the available goodreads chrome store extension support page.
Thanks for your work on this, it's so useful and it's implemented so much more elegantly than the other addon for carrying out this function.   As long as I'm here - a firefox addon would be so great!  I tried to convert it to firefox using 'Chrome Store Foxified' but alas, there must be some function you're using that's not supported.

**Comment by gyermich** [comment]
Looks like $ is now used by another library on the book description page, when looking at it in the console $("div#description") returns null, while jQuery("div#description") finds the correct div. Could be fixed by using [jQuery.noConflict](https://api.jquery.com/jQuery.noConflict/)

**Comment by rhollister** [comment]
Do you have an example library and book where this breaks? I wasn't able to reproduce it.

**Comment by gartkb** [comment]
I've tried it on over 13 libraries and I didn't find one that works.  Brooklyn public library being one that is broken.
I manually applied the change mentioned in the PR and it did fix the problem.  I've just confirmed the problem still exists on the existing version of the addon on a separate machine.

At the risk of being rude for hijacking here I'd just like to mention that the "request" feature is really handy, but it is also broken on every library I've tried except for one (currently all broken because of the above change, but returns in the PR applied version).  Works on vpl: http://vpl.lib.overdrive.com/BANGSearch.dll?Type=FullText&FullTextField=All&more=1&FullTextCriteria=Believing%20Bullshit%20Stephen%20Law

but not on any other what I think of as "new style" overdrive such as brooklyn: 
https://brooklyn.overdrive.com/search?query=Believing%20Bullshit%20Stephen%20Law

I can make this into a new issue if you like, just say the word.

By the way, I can't say enough good things about this addon.  It is so fantastically presented and is a real lifesaver.
